docs/specification.rst

```diff
@@ -693,9 +693,10 @@ A literal expression is an expression that allows arbitrary JSON objects to be
 specified.  This is useful in filter expressions as well as multi select hashes
 (to create arbitrary key value pairs), but is allowed anywhere an expression is
 allowed.  The specification includes the ABNF for JSON, implementations should
-use an existing JSON parser to parse literal values.  Note that the ``\```
-character must now be escaped in a ``json-value`` which means implementations
-need to handle this case before passing the resulting string to a JSON parser.
+use an existing JSON parser to parse literal values.  Note that the
+:literal:`\`` character must now be escaped in a ``json-value`` which means
```

Comment:
Unfortunately, the inline-literal markup (` ``...`` `) doesn't respect back-slash for escaping purposes while the [`:literal:` role](http://docutils.sourceforge.net/docs/ref/rst/roles.html#literal) does.